test/new-e2e/system-probe/test-json-review/testowners_test.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
+
+package main
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const ownersfile = `
+# A comment
+/pkg/network @networks
```

Comment:
Do you know what happens when there are multiple owners listed? I know we have that a lot.

Reply:
It takes all into account, added a test just in case